Tests/Marketplace/core_packs_list.json

```diff
@@ -1,32 +1,43 @@
-[
-  "Base",
-  "rasterize",
-  "DemistoRESTAPI",
-  "DemistoLocking",
-  "ImageOCR",
-  "WhereIsTheEgg",
-  "AutoFocus",
-  "UrlScan",
-  "Active_Directory_Query",
-  "FeedTAXII",
-  "VirusTotal",
-  "Whois",
-  "Phishing",
-  "CommonScripts",
-  "CommonPlaybooks",
-  "CommonTypes",
-  "CommonDashboards",
-  "CommonReports",
-  "CommonWidgets",
-  "TIM_Processing",
-  "TIM_SIEM",
-  "HelloWorld",
-  "ExportIndicators",
-  "Malware",
-  "DefaultPlaybook",
-  "AccessInvestigation",
-  "ThreatIntelligenceManagement",
-  "ipinfo",
-  "FeedMitreAttackv2",
-  "FeedUnit42v2"
-]
\ No newline at end of file
+{
```

Comment:
please use a separate file for it, as the SDK is also using it for validation.

Reply:
Done